packages/web/src/ssr/util.ts

```diff
@@ -61,3 +61,21 @@ export const makePageRoute =
     }
     return false
   }
+
+export const checkIsBot = (userAgent: string) => {
```

Comment:
Don't think we need this right now, but could be useful in the future! We can replace general admission with ssr client

Reply:
yea i just grabbed both bc they were right next to each other 👌